Corona-Warn-App/src/main/java/de/rki/coronawarnapp/appconfig/sources/fallback/DefaultAppConfigSource.kt

```diff
@@ -0,0 +1,31 @@
+package de.rki.coronawarnapp.appconfig.sources.fallback
+
+import android.content.Context
+import dagger.Reusable
+import de.rki.coronawarnapp.appconfig.ConfigData
+import de.rki.coronawarnapp.appconfig.internal.ConfigDataContainer
+import de.rki.coronawarnapp.appconfig.mapping.ConfigParser
+import de.rki.coronawarnapp.util.di.AppContext
+import org.joda.time.Duration
+import org.joda.time.Instant
+import javax.inject.Inject
+
+@Reusable
+class DefaultAppConfigSource @Inject constructor(
+    @AppContext private val context: Context,
+    private val configParser: ConfigParser
+) {
+
+    fun getRawDefaultConfig(): ByteArray {
+        return context.assets.open("default_app_config.bin").readBytes()
+    }
+
+    fun getConfigData(): ConfigData = ConfigDataContainer(
+        mappedConfig = configParser.parse(getRawDefaultConfig()),
+        serverTime = Instant.EPOCH,
+        localOffset = Duration.standardHours(1),
```

Comment:
Why is the default +1h? If we deal with time zones properly we would not need an offset, do we?

Reply:
Should we set it to 0?
I set it to something that is not "great" (0), but also not terrible (2+hours, which would once we validate the time offset, prevent key submission).

We could discuss whether using the default config for key submission would be acceptable? @mlenkeit  If it is the offset should be below the allowed time diff otherwise above.

Reply:
Offset from server to local time.

https://github.com/corona-warn-app/cwa-app-android/blob/a1d8465e2d988e83d56b1979ddbe5fabd85527de/Corona-Warn-App/src/main/java/de/rki/coronawarnapp/appconfig/download/AppConfigServer.kt#L64

Reply:
but then I would set it to zero for the default.

Reply:
Done.